commit d86e4049f58460fdf857484dbeb8de07a1a76d04
Author: Mike Snitzer <snitzer@redhat.com>
Date:   Thu Feb 16 23:57:17 2017 -0500

    dm round robin: revert "use percpu 'repeat_count' and 'current_path'"
    
    commit 37a098e9d10db6e2efc05fe61e3a6ff2e9802c53 upstream.
    
    The sloppy nature of lockless access to percpu pointers
    (s->current_path) in rr_select_path(), from multiple threads, is
    causing some paths to used more than others -- which results in less
    IO performance being observed.
    
    Revert these upstream commits to restore truly symmetric round-robin
    IO submission in DM multipath:
    
    b0b477c dm round robin: use percpu 'repeat_count' and 'current_path'
    802934b dm round robin: do not use this_cpu_ptr() without having preemption disabled
    
    There is no benefit to all this complexity if repeat_count = 1 (which is
    the recommended default).
    
    Signed-off-by: Mike Snitzer <snitzer@redhat.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1264_d86e4049f58460fdf857484dbeb8de07a1a76d04.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1264_d86e4049f58460fdf857484dbeb8de07a1a76d04.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
